audio/tauonmb/tauonmb.SlackBuild

```diff
@@ -27,8 +27,8 @@
 cd $(dirname $0) ; CWD=$(pwd)
 
 PRGNAM=tauonmb
-VERSION=${VERSION:-7.2.1}
-SRCNAM=TauonMusicBox
+VERSION=${VERSION:-7.8.0}
+SRCNAM=Tauon
 BUILD=${BUILD:-2}
```

Comment:
This should be reset to 1

Reply:
@willysr I'm gonna fix this later, could you please tell me what to do about [this](https://github.com/SlackBuildsOrg/slackbuilds/pull/6817#issuecomment-2177781763)?